src/util/GlobalChecks.cpp

```diff
@@ -38,6 +39,7 @@ printErrorAndAbort(const char* s1)
 {
     std::fprintf(stderr, "%s\n", s1);
     std::fflush(stderr);
+    printCurrentBacktrace();
```

Comment:
Can we inject this in the `std::terminate` chain as well (that invokes `std::abort`)?

Reply:
Maybe? I can look into it if you like but if you want it for the case where there's an uncaught exception, it's usually too late to get anything out of a backtrace -- we've already unwound.

I guess it also gets called in the case of double-faults / throws-during-catches which might be interesting to see. Is that the sort of thing you want to see?